.sequelizerc

```diff
@@ -0,0 +1,10 @@
+require("@babel/register")
+const path = require('path')
+require('dotenv').config();
+
+module.exports = {
+  config: path.resolve('src/database/config', 'config.js'),
+  'models-path': path.resolve('src/database/models'),
+  'seeders-path': path.resolve('src/database/seeders'),
+  'migrations-path': path.resolve('src/database/migrations'),
+}
```

Comment:
@hakoemmy  ESLint

Reply:
@AbdoulNuru  thank you. Allow me to fix this